lang/en/tool_mfa.php

```diff
@@ -34,21 +34,21 @@
 $string['weight'] = 'Weight';
 $string['settings'] = 'Settings';
 $string['action'] = 'Action';
-$string['edit'] = 'Edit';
 $string['revoke'] = 'Revoke';
 $string['created'] = 'Created';
-$string['modified'] = 'Modified';
+$string['createdfromip'] = 'Created from IP';
+$string['lastverified'] = 'Last verified';
 $string['addfactor'] = 'Add factor';
-$string['editfactor'] = 'Edit factor';
+$string['revokefactor'] = 'Revoke factor';
 $string['enablefactor'] = 'Enable factor';
-$string['disablefactor'] = 'Disable factor';
+$string['revokefactor'] = 'Revoke factor';
```

Comment:
duplicate string?

Reply:
as a general rule, its nice to keep the lang string files sorted and then it makes it easier to pick up issues like this.

For that reason it can also be helpful to have string keys which are 'object''actions', eg 'factorrevoke' 'factorenable' and then they are all grouped nicely. I'll make another card that that https://github.com/catalyst/moodle-tool_mfa/issues/51